static/index.html

```diff
@@ -100,7 +139,14 @@ <h1><a href="/" tabindex="0">linus.zone</a></h1>
           idInput.addEventListener('input', debounce(async () => {
               const showPasswordRequired = await checkIfShouldShowPasswordRequired();
               setPasswordGroupVisibility(showPasswordRequired);
+              await populateContentForExistingNote();
           }, 400));
+          uriInput.addEventListener('input', () => {
+              userTypedInput = true;
+          });
+          noteInput.addEventListener('input', () => {
```

Comment:
For both of these event listeners, I think it would be good to have `userTypedInput = false` again when the inputs are completely erased. It's a weird UX to type something, erase it fully, and still be prompted on whether the existing input can be wiped.